x/ibc/light-clients/solomachine/types/client_state_test.go

```diff
@@ -24,56 +24,60 @@ var (
 )
 
 func (suite *SoloMachineTestSuite) TestClientStateValidateBasic() {
-	testCases := []struct {
-		name        string
-		clientState *types.ClientState
-		expPass     bool
-	}{
-		{
-			"valid client state",
-			suite.solomachine.ClientState(),
-			true,
-		},
-		{
-			"empty ClientState",
-			&types.ClientState{},
-			false,
-		},
-		{
-			"sequence is zero",
-			types.NewClientState(0, &types.ConsensusState{suite.solomachine.ConsensusState().PublicKey, suite.solomachine.Diversifier, suite.solomachine.Time}, false),
-			false,
-		},
-		{
-			"timestamp is zero",
-			types.NewClientState(1, &types.ConsensusState{suite.solomachine.ConsensusState().PublicKey, suite.solomachine.Diversifier, 0}, false),
-			false,
-		},
-		{
-			"diversifier is blank",
-			types.NewClientState(1, &types.ConsensusState{suite.solomachine.ConsensusState().PublicKey, "  ", 1}, false),
-			false,
-		},
-		{
-			"pubkey is empty",
-			types.NewClientState(1, &types.ConsensusState{nil, suite.solomachine.Diversifier, suite.solomachine.Time}, false),
-			false,
-		},
-	}
+	// test singlesig and multisig public keys
+	for _, solomachine := range []*ibctesting.Solomachine{suite.solomachine, suite.solomachineMulti} {
```

Comment:
I add this for loop into each test and changed all variables from `suite.solomachine` to `solomachine`. The diffs don't reflect this well